lira/parsers/rst.py

```diff
@@ -22,13 +23,22 @@ class DirectiveNode(Element):
     - content
     - options
     """
-
     tagname = "directive"
 
 
-def importable(value):
-    # TODO: check if value can be imported
-    return value
+def is_importable(value):
+    module_name, class_name = value.rsplit(".",1)
+    try:
+        MyClass = getattr(importlib.import_module(module_name), class_name, None)
+        instance = MyClass()
```

Comment:
We don't need to create an instance (I think), we just need to check if the class is a subclass of `lira.validators.Validator`